cmd/genji/shell/command.go

```diff
@@ -2,20 +2,30 @@ package shell
 
 import (
 	"fmt"
-	"sort"
+	"os"
 
+	"github.com/agnivade/levenshtein"
 	"github.com/genjidb/genji"
 	"github.com/genjidb/genji/document"
 )
 
-var commands = map[string]string{
-	".tables":               "\t\tList names of tables.",
-	".exit":                 "\t\t\tExit this program.",
-	".indexes [table_name]": "\tDisplay all indexes or the indexes of the given table name.",
-	".help":                 "\t\t\tList all commands.",
+type Command struct {
+	Name    string
+	Options string
+	Help    string
+	Action  func(db *genji.DB, cmd []string) error
 }
 
-func runTablesCmd(db *genji.DB, cmd []string) error {
+const TotalCommands = 4
```

Comment:
Creating globals must be done for a very good reason in my opinion, especially when they are exported.